packages/core/shared/src/nodes/text/GenerateText.ts

```diff
@@ -179,7 +179,7 @@ export class GenerateText extends MagickComponent<Promise<WorkerReturn>> {
       throw new Error('ERROR: Completion handler undefined')
     }
 
-    const { success, result, error } = await completionHandler({
```

Comment:
The completion handlers this is calling are async, so I think we need this await